erddap/content/datasets.xml

```diff
@@ -2152,8 +2212,10 @@
       </sourceAttributes -->
       <addAttributes>
         <att name="_FillValue" type="float">NaN</att>
-        <att name="ioos_category">Unknown</att>
-        <att name="long_name">S27 Temp</att>
+        <att name="ioos_category">Temperature</att>
+        <att name="long_name">S27 Vertically Averaged Sea Water Temperature</att>
+        <att name="standard_name">integral_wrt_depth_of_sea_water_temperature</att>
+        <att name="units">1</att>
```

Comment:
The unit here should, likely, be degree_C

Reply:
Same comment as first comment applies, happy to make this change, either way if it's a mathematical index i.e. not the actual temperature, or the deviation from the climate norm.